torchmetrics/metric.py

```diff
@@ -28,6 +30,10 @@
 from torchmetrics.utilities.imports import _LIGHTNING_AVAILABLE, _compare_version
 
 
+def is_distributed_fn() -> bool:
```

Comment:
prefer this name but no strong opinion

```suggestion
def distributed_available() -> bool:
```

would need to update the other instances

Reply:
Done !